github/event_types.go

```diff
@@ -130,17 +130,6 @@ type GollumEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
-// EditChange represents the changes when an issue, pull request, or comment has
-// been edited.
-type EditChange struct {
-	Title *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"title,omitempty"`
-	Body *struct {
-		From *string `json:"from,omitempty"`
-	} `json:"body,omitempty"`
-}
```

Comment:
In that case, @varadarajana please revert the changes to other structs for now, so we can further discuss just the structs needed for the new functionality being added as part of this PR only. That'll make it easier.